internal/plugin.go

```diff
@@ -844,17 +844,27 @@ func (p *Plugin) createPolicy(policyConf *types.PolicyConfig) error {
 		return err
 	}
 
+	annotations := map[string]string{
+		"policy.open-cluster-management.io/categories": strings.Join(policyConf.Categories, ","),
+		"policy.open-cluster-management.io/controls":   strings.Join(policyConf.Controls, ","),
+		"policy.open-cluster-management.io/standards":  strings.Join(policyConf.Standards, ","),
+	}
+
+	for k, v := range p.PolicyDefaults.PolicyAnnotations {
```

Comment:
To match the rest of the code, could you apply the defaults on the policy configuration in the `applyDefaults` method?

Reply:
done